src/main/kotlin/by/jprof/telegram/opinions/processors/YoutubeLinksProcessor.kt

```diff
@@ -98,12 +98,13 @@ class YoutubeLinksProcessor(
                     logger.debug("$channelId is in a white list")
                     val videoText = "${"Vote for video: $youtubeLink".boldMarkdownV2()} \n$description " +
                             "\nLikes: $likes Dislikes: $dislikes".boldMarkdownV2() //trim indent have strange layout
+                    val votes = votesDAO.get(videoId) ?: Votes(videoId)
```

Comment:
Consider adding a prefix for all votes by this processor, just like the JEP one does. So that votes can be distinguished. Of course, YouTube links are unique, but if the number of processors grows… Votes should have prefixes to be globally unique. Imagine having a JEP and JSR processors? Both have IDs in form of numbers. How do we distinguish votes then?

```suggestion
                    val votes = votesDAO.get("YOUTUBE"+videoId) ?: Votes("YOUTUBE"+videoId)
```

Reply:
I was uncertain about that prefix actually, I'll add one